src/loaders/relocate-loader.js

```diff
@@ -558,28 +557,26 @@ module.exports = function (code) {
             transformed = true;
             magicString.overwrite(staticChildNode.start, staticChildNode.end, replacement);
           }
-          staticChildNode = staticChildValue = undefined;
         }
-        else if (stats && stats.isDirectory() &&
-            // dont emit __dirname or package base
-            staticChildValue !== path.dirname(id) &&
-            staticChildValue !== pkgBase) {
+        else if (stats && stats.isDirectory()) {
           let replacement = emitAssetDirectory(path.resolve(staticChildValue));
           if (replacement) {
             transformed = true;
             magicString.overwrite(staticChildNode.start, staticChildNode.end, replacement);
           }
-          staticChildNode = staticChildValue = undefined;
         }
+        staticChildNode = staticChildValue = undefined;
```

Comment:
This was the bug (1) in the directory emission that was stopping further directory emissions from working after passing the line above.